src/pages/e-ticket.tsx

```diff
@@ -0,0 +1,37 @@
+import Image from 'next/image';
+import Link from 'next/link';
+
+// using https://github.com/dephraiim/qrcode
+// example: https://qr-service.vercel.app/api/code?text=https://freshmen2023.sgcu.in.th/&size=200
+
+const ETicket = () => {
+    return (
+        <div className="min-h-screen">
+            <div>
+                <Link href="https://freshmen2023.sgcu.in.th/profile">
+                    <button className="justify-left relative h-10 w-10 rounded-lg px-16 py-16">
```

Comment:
dont have this class in tailwindcss

Reply:
Please avoid using `relative` and use flex box to align them instead